common/src/storage/complex/mailbox.rs

```diff
@@ -99,6 +99,11 @@ pub trait MailboxError {
     fn element_not_found() -> Self;
 }
 
+impl MailboxError for () {
```

Comment:
Add comment this is for mock only

Reply:
Pretty obvious cause its the same pattern for anything substrate-related